tests/bsim/bluetooth/audio/src/cap_acceptor_test.c

```diff
@@ -952,14 +923,13 @@ static void test_cap_acceptor_broadcast(void)
 
 	printk("Scan stopped, attempting to PA sync to the broadcaster with id 0x%06X\n",
 	       broadcaster_broadcast_id);
-	err = pa_sync_create();
-	if (err != 0) {
-		FAIL("Could not create Broadcast PA sync: %d\n", err);
-		return;
-	}
 
-	printk("Broadcast source found, waiting for PA sync\n");
-	WAIT_FOR_FLAG(flag_pa_synced);
+	pa_sync_create();
+}
+
+static void bap_sink_stream(struct bt_bap_stream *bap_streams[], size_t *stream_count)
+{
```

Comment:
That's odd function name :) Suggest to rename it to something more fitting that also explains what it does